src/Razor/src/Microsoft.CodeAnalysis.Remote.Razor/Remote/ServiceDescriptors.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using Microsoft.ServiceHub.Framework;
+
+namespace Microsoft.CodeAnalysis.Remote.Razor
```

Comment:
This is copied from Roslyn and is designed for multiple services. it can be simplified if Razor continues to have just a single service.